src/test/java/BaseTest.java

```diff
@@ -1,7 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+
+import java.time.Duration;
 
 public class BaseTest {
 
+    static WebDriver driver;
+    static String newPlaylistName = "Nozima's Songs";
```

Comment:
try to avoid static fields unless you clearly understand why you need it. The fact is that you can accidentally call this variable when there is a big real project. In the future, most likely you will use Spring Boot and autowaried annotations do not work with statics, this will be a big problem